core-features/napi-preview-feature/prisma/schema.prisma

```diff
@@ -0,0 +1,23 @@
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+generator client {
+  provider        = "prisma-client-js"
+  binaryTargets = ["native"]
```

Comment:
setting "native" might have different effects than not setting anything maybe? so probably worth _another_ test or leaving it off here to be more what a normal user would do.

Reply:
Removed